Makefile

```diff
@@ -104,10 +104,11 @@ addman: help2man man-db
 	mandb
 
 version:
+	mkdir -p /var/lib/dokku
```

Comment:
This should use the `DOKKU_LIB_ROOT` variable (same elsewhere in the makefile).

Reply:
updated